### DISCUSSION TITLE
Very weird formatting

### DISCUSSION THREAD

**Discussion by darkBuddha** [danny-avila/LibreChat#4337] [Troubleshooting]
### What happened?

LibreChat makes parts of the prompt cursive, bold, crossed out for no reason. On simple text prompts, bash scripts, I could not detect any pattern. When selecting parts of the user-sent prompt, and copy/pasting, it looks even crazier, with newlines after each character.

Not related to LaTeX, tried with and without, exactly the same.

### Steps to Reproduce

1. Post anything into a LibreChat v0.7.5-rc2 prompt, e.g.: 

```
darkBuddha@unifi:~$ sudo apt update
Hit:1 http://security.debian.org/debian-security bookworm-security InRelease
Hit:2 https://deb.debian.org/debian-security bookworm-security InRelease                                                                                                    
Hit:3 https://deb.debian.org/debian bookworm InRelease                                                                                                                      
Ign:4 https://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 InRelease                                                                                                
Hit:5 http://ftp.halifax.rwth-aachen.de/debian bookworm InRelease                                                                                                           
Hit:6 http://ftp.halifax.rwth-aachen.de/debian bookworm-updates InRelease                                                                
Hit:7 https://apt.glennr.nl/repo bookworm InRelease                                                                                      
Get:8 https://packages.adoptium.net/artifactory/deb bookworm InRelease [7,517 B]          
Hit:9 https://dl.ui.com/unifi/debian unifi-8.4 InRelease                        
Hit:10 https://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 Release
Fetched 7,517 B in 1s (8,414 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
1 package can be upgraded. Run 'apt list --upgradable' to see it.
darkBuddha@unifi:~$ sudo apt upgrade
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  unifi
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
darkBuddha@unifi:~$ 
```

### What browsers are you seeing the problem on?

Google Chrome macOS

### Screenshots

![Screenshot 2024-10-04 at 11 29 26](https://github.com/user-attachments/assets/fca3117a-83f7-4011-b43d-d40880953f3c)

After copy/paste:

![Screenshot 2024-10-04 at 11 29 47](https://github.com/user-attachments/assets/7d709187-1fd8-489c-a1e7-b591fefc4c44)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
User messages are now being rendered as markdown by default. Markdown has some specific rules about syntax and even indenting can affect the format.

To "optimize" the format, I would wrap your terminal logs in markdown code blocks:

~~~markdown
please review this:
```
your terminal logs
```
~~~

Or, you can disable this behavior: 

![image](https://github.com/user-attachments/assets/08adb25a-7cb3-4b4d-8f75-c56ecb42dc0f)

**Comment by darkBuddha** [comment]
thx for the clarification!
